.github/workflows/beta-release.yml

```
@@ -388,6 +456,135 @@ jobs:
             apps/frontend/dist/*.exe
             apps/frontend/dist/*.yml
 
+  # Windows ARM64 build on ARM64 runner for native compilation
+  build-windows-arm64:
+    needs: create-tag
+    runs-on: windows-arm64-4-core
```

actionlint (1.7.10)

label "windows-arm64-4-core" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-xlarge", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)